services/QuillLMS/engines/evidence/app/services/evidence/gen_ai.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Evidence
+  module GenAI
+    HistoryItem = Struct.new(:user, :assistant, keyword_init: true)
```

Comment:
Maybe an opportunity to use the new value object interface: `Data.define(:user,...)`

Reply:
Definitely, updating!